DuplexMaker.py

```diff
@@ -106,7 +106,7 @@ def main():
     firstTag=firstRead.qname.split(":")[0]
     qualScore = firstRead.qual # Set a dummy quality score
     consensusDict={}
-    cigDum = "%dM" % o.read_length #set a dummy cigar score
+    cigDum = firstRead.cigar #set a dummy cigar score
```

Comment:
Ugh, sorry, this is not a string.  I still think that `a.flag = 5` should be set.